src/graphapi/graphapi.cpp

```diff
@@ -139,6 +144,18 @@ extern "C" miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t desc
 
         switch(descriptorType)
         {
+        case MIOPEN_BACKEND_POINTWISE_DESCRIPTOR:
+            if(std::align(alignof(miopen::graphapi::BackendPointwiseDescriptor),
```

Comment:
you may want to refactor it into a single function that works for all descriptor types. Less repetition that way.

Reply:
Let's take this into the next refinement PR :)

Reply:
Done in https://github.com/ROCm/MIOpen/pull/2792